project.mml

```diff
@@ -1663,6 +1666,8 @@ Layer:
             OR tags->'memorial' IN ('plaque')
             OR military IN ('bunker')
             OR highway IN ('bus_stop', 'elevator', 'traffic_signals')
+            OR tags->'public_transport' IN ('platform')
+            OR tags->'bus' IN ('yes')
```

Comment:
Not needed, we don't render bus=yes without public_transport=platform

Reply:
What would you propose to do instead?

Reply:
Remove the line. You could further restrict the public_transport condition to only get those with bus=yes, but that makes no difference to the rendering